tests/02_00_05-2wp_version1.js

```diff
@@ -106,188 +98,6 @@ describe('Lock funds using peg-in protocol version 1', () => {
         expect(Number(finalDestinationAddressBalance)).to.equal(Number(btcEthUnitConverter.btcToWeis(AMOUNT_TO_LOCK_IN_BTC)));
     });
 
-    it('should refund to indicated address when locking cap exceeded using bech32 sender', async () => {
```

Comment:
Shouldn't we test that the locking doesn't exist anymore?

Reply:
Why test something that doesn't exist?